contracts/V3_CHANGELOG.md

```diff
@@ -21,3 +21,4 @@ _This document is intended to document and explain the Art Blocks Core V3 change
     - `projectScriptDetails` - Information relevant to rendering tokens
     - `projectMetadata` - Information relevant to understanding the project as a work of art
     - `projectArtistPaymentInfo` - Information relevant to artists as they manage their primary and additional payment accounts
+- Never allow an increase in Project edition size
```

Comment:
May be worth adding a note here in the changelog, or filing a ticket internally, but we should follow up pre-launch with an update to our creator docs making it really aggressively explicit that this is the new behavior to avoid any artist confusion on the matter.

Reply:
Thanks for the callout - I've added an internal ticket about communicating this (and any other important V3 details).